jekyll/_cci2/migrating-from-teamcity.adoc

```diff
@@ -0,0 +1,454 @@
+= Migrating from TeamCity 
+:page-layout: classic-docs
+:page-liquid:
+:source-highlighter: pygments.rb
+:icons: font
+:toc: macro
+:toc-title:
+:sectanchors:
+
+This document provides an overview of how to migrate from TeamCity to CircleCI.
+
+== Quick Start
+
+CircleCI has various differences in setup and configuration from TeamCity. For the best experience of trying CircleCI, we recommend getting started with the following:
+
+. **Read (or at least browse) this document** to solidify conceptual differences.
+. **Quickly build and run a simple https://circleci.com/docs/2.0/hello-world/#section=getting-started[Hello World job] in any repository**. This configuration demonstrates the simplicity of our YAML setup, and it showcases how to define both an execution environment and the steps to run within it. All jobs on CircleCI share this format.
+. **Once you understand the config, modify the executor and the individual steps to your needs.** For example, a https://circleci.com/docs/2.0/language-ruby/[Rails project] may use a Ruby container and run `bundler` commands. Consult our https://circleci.com/docs/2.0/tutorials/#section=configuration[Tutorials page] for various examples in different languages and frameworks.
+. **After finishing one job, setup your other jobs on CircleCI and then https://circleci.com/docs/2.0/workflows/[configure a workflow]**. This would essentially migrate your Build Chain.
+. **Next, https://circleci.com/docs/2.0/optimizations/[optimize your jobs] with our advanced features** to see how much faster your builds could run on our platform.
+. **Throughout this process, consult our https://circleci.com/docs/2.0/configuration-reference/#section=configuration[Config Reference]** for a list of all available configuration on our platform.
+
+== Why Migrate to CircleCI?
+
+CircleCI is a modern CI/CD tool aimed at performance and developer productivity. With a flexible system for configuration, you can define complex workflows and jobs for your application's specific needs.
+
+**Configuration As Code**:: The majority of CircleCI's configuration is done via YAML. It is easy to maintain and trivial to learn, and your CI/CD pipeline can be tracked and versioned like the rest of your source code.
```

Comment:
Just wondering what the significance of double "::" is?

Reply:
Ah oops that's a typo.

Reply:
@rosieyohannan Oh actually, no it's not a typo. The double colons are description lists in ASCII Doc: https://asciidoctor.org/docs/asciidoc-writers-guide/#description-lists

Reply:
Oh great! :-D